### ISSUE TITLE
As an applicant, the status of my application shows 'submitted' after being approved, and I cannot see that it has been approved. Nor can I accept/decline.

### ISSUE THREAD

**Issue by YorgunKelebek** [goofballLogic/g4a-spa#19]

**Comment by goofballLogic** [comment]
Yes I thought it might be better to keep approve/decline as a distinct piece of functionality which would probably be initiated by e-mail? So in other words it wouldn't happen via the applicant portal at all. 

In the future it probably makes sense to have an area where grant recipients can do things like file reports on how they are using money and the accept/decline functionality could live in that location? Or even there could be a distinct page just for accept/decline as it's sort of a special period in the grant managers workflow which requires input from the applicant.

**Comment by YorgunKelebek** [comment]
OK, agreed - for now it could remain as off-system with the ability for the manager to manually approve/decline on behalf of the applicant.

Status of the application should be updated on the portal for the applicant to see though.

**Comment by goofballLogic** [comment]
Why does the portal needs to reflect that?

**Comment by YorgunKelebek** [comment]
If you were an applicant, when you look at your application on the portal would you not want the status of your application to reflect where it is in the workflow accurately?

**Comment by goofballLogic** [comment]
I suppose it would depend on whether or not I had Eg received an email with a link to confirm receipt with a link to another page where I could check status?

**Comment by goofballLogic** [comment]
Basically the applicant wouldn't see any update on the status until approved or denied at which point they would receive an email notification anyway ,(I'm assuming)

**Comment by YorgunKelebek** [comment]
> Basically the applicant wouldn't see any update on the status until approved or denied

Yes, this is exactly what I mean - email is never 100% reliable, which may go into spam etc. so it would be neat to see the status on the portal...

However, I have remembered that the status the applicant see is coming from a copy of the grant document they have, which is submitted, not the one that has been reviewed and therefore approved/rejected... That still does not justify it in my opinion though. Architectural decision such as this should not drive the functionality that should be made available or not!

**Comment by goofballLogic** [comment]
So the reason is that the applicant might want to check the status themselves in the event that email didn't arrive?

**Comment by YorgunKelebek** [comment]
> So the reason is that the applicant might want to check the status themselves in the event that email didn't arrive?

Yes, I would imagine they would.

**Comment by goofballLogic** [comment]
Ok i think what might work well is a new page for a submitted application where the applicant can do things like accept an offer too.
